terraform/authentication/env/dev/example_tfvars.txt

```diff
@@ -0,0 +1,7 @@
+aws_region     = "eu-west-2"
+app_name       = "tech-radar"
+environment    = "dev"
+container_port = 80
+cpu            = 256
```

Comment:
Double check cpu and memory constraints are as low as possible (start by looking at what policy dashboard is using). This affects cost.

Reply:
Reduced to 512 CPU; 1024 Mem